fixed typo in first line

mas-designs opened the pull request:
There was becuase instead of because!

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

mas-designs committed:
fixed typo in first line

There was becuase instead of because!

b3u commented:
LGTM

huyenltnguyen commented on guide/english/jquery/jquery-click-method/index.md:
```diff
@@ -5,7 +5,7 @@ title: Click Method
 
 
-The jQuery Click method triggers an function when an element is clicked. The function is known as a "handler" becuase it handles the click event. Functions can
+The jQuery Click method triggers an function when an element is clicked. The function is known as a "handler" because it handles the click event. Functions can
```
Hi @mas-designs, thank you for the PR. Your change looks good to me. However, there is another typo/grammar error in the first sentence:
```
The jQuery Click method triggers an function...
```
should be
```
The jQuery Click method triggers a function...
```

Would you mind fixing it as well?

mas-designs committed:
Removed second typo.

I've removed the second type as requested, thank you I've totally not seen this at first!

mas-designs commented:
@huyenltnguyen thank you for mentioning it! Totally missed this, I've updated the file, please let me know if there is anything else I should do!

huyenltnguyen commented:
@mas-designs Thank you for the additional change! Your PR looks good to me. Unfortunately, it is not ready to merge at the moment due to the Travis CI build failure. This is totally not your fault. I think there was a recent change to the repo that caused it, and many PRs have been affected as well.

I will check with other moderators and try to get your PR merged ASAP.

huyenltnguyen commented:
Awesome! We have those green ticks back! I'm merging it now.

@mas-designs Thank you for the PR and congratulations on your first contribution to this repo ✨ 🎉 
Happy coding!
